content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -58,6 +58,10 @@ corresponding reclamation or reconciliation process, as a given admission
 controller does not know for sure that a given request will pass all of the
 other admission controllers.
 
+The ordering of these calls can be seen below.
+
+{{< figure src="/images/docs/admission-controllers/kube-apiserver.svg" alt="Sequence diagram for kube-apiserver handling requests during the admission phase showing mutation webhooks, followed by validatingadmissionpolicies and finally validating webhooks." class="diagram-large" >}}
+
```

Comment:
Suggestion: move the alt text out into the paragraph before the diagram.
The alt can be short one.

Reply:
What wording would you prefer for the alt-text? Don't want accessibility to take a hit
I can get this updated once you let me know!

Reply:
I like the alt attribute as it is, or maybe something _longer_. Short alt attributes may not help people much if they are have trouble seeing.

Reply:
Happy to redo this now that it's a mermaid diagram. How can I add alt-text to it?

Reply:
Try `accDescr`; see https://mermaid.js.org/config/accessibility.html

Reply:
Updated, is this better?